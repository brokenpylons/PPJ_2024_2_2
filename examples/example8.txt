var sum = 0;
for (var i = 0, 5) {
    sum = sum + i;
}

print(sum);
