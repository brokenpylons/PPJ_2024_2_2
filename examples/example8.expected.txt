define("var") variable("sum") assign("=") real("0") term(";") for("for") lparen("(") define("var") variable("i") assign("=") real("0") to(",") real("5") rparen(")") begin("{") variable("sum") assign("=") variable("sum") plus("+") variable("i") term(";") end("}") print("print") lparen("(") variable("sum") rparen(")") term(";") 
